src/domain/community/contributor/Account/AccountResourcesView.tsx

```
@@ -0,0 +1,165 @@
+          <ScrollableCardsLayoutContainer containerProps={{ flex: 1 }}>
+            {accountResources.spaces?.slice(0, visibleSpacesCount).map(contributionItem => (
+              <JourneyTile
+                journey={{
+                  profile: {
+                    displayName: contributionItem.profile.displayName,
+                    url: contributionItem.profile.url,
+                    cardBanner: contributionItem.profile.cardBanner,
+                  },
+                }}
+                journeyTypeName="space"
+              />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)